config/application.rb

```diff
@@ -11,6 +11,7 @@ class Application < Rails::Application
   	include Cul::Omniauth::FileConfigurable
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
+    config.autoload_paths += %W(#{config.root}/lib)
```

Comment:
Apparently I also have this set up in AC. I'll look into this more later, because I am curious. But we can leave it in for now.